sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -198,7 +198,9 @@ object QueryCompilationErrors {
   }
 
   def pandasUDFAggregateNotSupportedInPivotError(): Throwable = {
-    new AnalysisException("Pandas UDF aggregate expressions are currently not supported in pivot.")
```

Comment:
This message is changed from `"Pandas UDF aggregate expressions are currently not supported in pivot."` to `"Pivot does not support Pandas UDF aggregate expressions."` as suggested in https://spark.apache.org/error-message-guidelines.html.